src/makeCalculator.js

```diff
@@ -37,7 +37,28 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add: (x, y) => x + y,
+    subtract: (x, y) => x - y,
```

Comment:
Subtract method should only take one argument and subtract it from the current result.